.gitattributes

```diff
@@ -1 +1,2 @@
 evals/registry/data/**/*.jsonl filter=lfs diff=lfs merge=lfs -text
+*.jsonl filter=lfs diff=lfs merge=lfs -text
```

Comment:
Please revert your changes to this file.

Reply:
That was an unintended edit which I already reverted back to the original state as found here: https://github.com/openai/evals/blob/main/.gitattributes

Reply:
Appreciate it, thank you.